kitchen.yml

```diff
@@ -0,0 +1,208 @@
+---
+driver:
+  name: docker
+  use_sudo: false
+  privileged: true
+  provision_command: mkdir -p /run/sshd
+  run_command: /lib/systemd/systemd
+
+platforms:
+  # Latest distros
+  - name: debian-9
+  - name: ubuntu-18.04
+  - name: centos-7
+  - name: fedora-29
+  - name: opensuse-42.3
+    driver:
+      run_command: /usr/lib/systemd/systemd
+      provision_command:
+        - systemctl enable sshd.service
+
+  # Previous distros
+  - name: debian-8
+  - name: ubuntu-16.04
+  - name: fedora-28
+  # Fails on Debian due to vsyscall issues, see
+  # https://hub.docker.com/_/centos, "A note about vsyscall"
```

Comment:
Are you talking about Debian or Centos?

Reply:
Need to elaborate these comments better, atm they're my notes during the attempts to get this working: it means `centos-6` *guest* has issues on `debian` *hosts*, due to Debian having disabled `vsyscall`.

Reply:
Re-worded the comment to make it a bit more clear.